lib/streamlit/delta_generator.py

```diff
@@ -116,6 +117,10 @@
 Value = TypeVar("Value")
 DG = TypeVar("DG", bound="DeltaGenerator")
 
+# Type aliases for Parent Block Types
+BlockType = str
+ParentBlockTypes = Iterable[Tuple[BlockType, Optional[int]]]
```

Comment:
Why don't we have type alias for the second element in the tuple?

Reply:
I had a conflicting name with `RootContainer`, however I've just updated the PR and introduced `RootContainerType` variable 👍